lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -3824,7 +3824,7 @@ namespace Js
         }
     }
 
-    // Array.prototype.indexOf as defined in ES6.0 (final) Section 22.1.3.11
+    // Array.prototype.indexOf as defined in ES6.0 (final) Section 22.1.3.11 https://tc39.github.io/ecma262/2017/#sec-array.prototype.includes
```

Comment:
>ES6.0 (final) [](start = 45, length = 13)

nit: ES6 was ES 2015 ([https://www.ecma-international.org/ecma-262/6.0/](https://www.ecma-international.org/ecma-262/6.0/)) -- if you intend this reference to refer specifically to ES 2017 spec, also update this text to say ES 2017